src/components/dialogs/input-hooks.js

```diff
@@ -283,18 +284,30 @@ export const useConnectivityValue = ({
     }, [inputForm.toggleClear]);
 
     useEffect(() => {
-        setConnectivity({
-            voltageLevel: voltageLevelIdDefaultValue
-                ? voltageLevelOptions.find(
-                      (value) => value.id === voltageLevelIdDefaultValue
-                  )
-                : null,
-            busOrBusbarSection: busOrBusbarSectionIdDefaultValue
-                ? {
-                      id: busOrBusbarSectionIdDefaultValue,
-                  }
-                : null,
-        });
+        if (fetchedVoltageLevelOptions) {
+            fetchedVoltageLevelOptions.then((values) =>
+                setVoltageLevelOptions(
+                    values.sort((a, b) => a.id.localeCompare(b.id))
+                )
+            );
+        }
+    }, [fetchedVoltageLevelOptions]);
+
+    useEffect(() => {
+        if (voltageLevelOptions) {
```

Comment:
Maybe prefer to do
`if(!voltageLevelOptions) return;`
to avoid savage indentation in the following code (dedication @laurgarn )